tests/unit/metadata/test_table.py

```diff
@@ -136,124 +136,92 @@ def test__get_faker_method_pass_args(self):
-        metadata_dict = {
-            'fields': {
-                'foo': {
-                    'type': 'categorical',
-                    'pii': True,
-                    'pii_category': 'email'
-                }
+        metadata = Mock()
+        metadata._ANONYMIZATION_MAPPINGS = {}
+        foo_metadata = {
+            'type': 'categorical',
+            'pii': True,
+            'pii_category': 'email',
+        }
+        metadata._fields_metadata = {
+            'foo': foo_metadata,
+            'bar': {
+                'type': 'categorical',
+            },
+            'baz': {
+                'type': 'id',
             }
         }
+        foo_values = ['test1@example.com', 'test2@example.com', 'test3@example.com']
         data = pd.DataFrame({
-            'foo': ['test1@example.com', 'test2@example.com', 'test1@example.com']
+            'foo': foo_values,
+            'bar': ['a', 'b', 'c'],
+            'baz': [1, 2, 3],
         })
-        metadata = Table.from_dict(metadata_dict)
 
         # Run
-        metadata._make_anonymization_mappings(data)
-        foo_mappings = metadata._ANONYMIZATION_MAPPINGS[id(metadata)]['foo']
+        Table._make_anonymization_mappings(metadata, data)
+        foo_mappings = metadata._ANONYMIZATION_MAPPINGS.get(id(metadata), {}).get('foo')
 
         # Assert
-        assert len(foo_mappings) == 2
-        assert list(foo_mappings.keys()) == ['test1@example.com', 'test2@example.com']
-
-    def _mock_faker_getattr(obj, fn_name):
-        if fn_name == 'company':
-            return lambda: obj.__lang__
-        else:
-            return getattr(obj, fn_name)
+        assert mock_table._get_fake_values.called_once_with(foo_metadata, 3)
+        assert len(foo_mappings) == 3
+        assert list(foo_mappings.keys()) == foo_values
 
-    @patch('faker.generator.getattr', _mock_faker_getattr)
-    def test__make_anonymization_mappings_multiple_localizations(self):
-        """Test that the ``_make_anonymization_mappings`` method takes localizations into account
-        when creating mappings for anonymized values.
+    @patch('sdv.metadata.Table')
+    def test__make_anonymization_mappings_unique_faked_value_in_field(self, mock_table):
+        """Test that ``_make_anonymization_mappings`` method creates mappings for anonymized values.
 
-        The ``_make_anonymization_mappings`` method should create localized fake values if
-        `pii_locales` is specified in the metadata description.
+        The ``_make_anonymization_mappings`` method should map equal values in the original data
+        to the same faked value.
 
         Input:
         - DataFrame with a field that should be anonymized based on the metadata description.
-        Metadata description specifies multiple localizations using `pii_locales`.
         Output:
-        - The mappings created from the original values to localized faked values.
+        - The mappings created from the original values to faked values.
```

Comment:
I think this should be marked as a `Side Effect` rather than `Output`, since the method does not really return anything